src/client/pages/forum/Post.jsx

```diff
@@ -33,41 +51,52 @@ const Post = (props) => {
 					<div className="card-footer">
 						<div className="card-user">
 							<img
-								src={post.user.profile.profilePicture}
+								src={!post.user.profile ? './assets/default-user.jpg' : post.user.profile["profilePicture"]}
```

Comment:
There is a default profile picture already assigned in the database so this image check isn't needed.

Reply:
Ok, Thanks